index.bs

```diff
@@ -477,9 +481,16 @@ The IAM market is thriving, with an estimated growth of 43 billion USD in 2029 [
 
 ### Global Workforce ### {#global-workforce}
 
-Digital Transformation has been a trend for several years and has played a crucial role, particularly in the Workforce, during the COVID-19 pandemic. Organizations could decide whether to stop operations or change as far as possible by digitizing and enabling remote work. This transformation made it clear that looking into the global workforce is possible.
+Digital Transformation has been a trend for several years and has played a crucial role, particularly in the Workforce, during the COVID-19 pandemic. 
+Organizations could decide whether to stop operations or change as far as possible by digitizing and enabling remote work. 
+This transformation made it clear that looking into the global workforce is possible.
```

Comment:
```suggestion
Because remote work implies fewer geography-based constraints, there will be demands for other forms of identification, and for interoperable credentials, in order to meet the demands of a more mobile workforce.
```